### ISSUE TITLE
Remove an unused parameter from the select reactor

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6773]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#6773|
|Type|enhancement|
|Created|2013-10-08 05:27:25Z|

It looks like it's the remnant of some old code, but now it doesn't do anything.

Attachments:

* [t6773.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/f13/f1345e5e4f6d41e65c3d3d65d716c6d149c5c9f2/aab97f1003593eda45e41be95160ddac69798fde.diff) (778 bytes) - added by Alex on 2013-10-08 05:27:41Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6773 6773
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1381210045000000 1381210045000000
changetime__1381282345000000 1381282345000000
version__None None
owner__therve therve

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

> It looks like it's the remnant of some old code, but now it doesn't do anything.

Notice there is no local `dict` in the calling code. :)

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve set owner to @therve|
|-|-|

Looks good, applying.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set status to `closed`|
|-|-|

(In [40198]) Author: Alex
Reviewer: therve
Fixes: [#6773](https://github.com/twisted/twisted/issues/6773)

Remove unused argument in select reactor fd dispatch logic.
